src/test/java/com/google/sps/data/DatastoreScheduledInterviewTest.java

```diff
@@ -184,4 +195,46 @@ public void failsGetScheduledInterview() {
     Optional<ScheduledInterview> actual = dao.get(2);
     Assert.assertEquals(Optional.empty(), actual);
   }
+
+  // Tests retrieving all scheduledInterviews for a particular user in a certain range
+  @Test
+  public void getsScheduledInterviewsInRange() {
+    dao.create(scheduledInterview1);
+    dao.create(scheduledInterview2);
+    dao.create(scheduledInterview3);
+    dao.create(scheduledInterview4);
+    List<ScheduledInterview> result =
+        dao.getScheduledInterviewsInRangeForUser(
+            "user@company.org",
```

Comment:
Please use `scheduledInterview1.interviewerEmail()` to make it clearer that this person was in the list above.

Reply:
Done